vault/src/main.rs

```diff
@@ -32,6 +32,8 @@ async fn start() -> Result<(), Error> {
     let opts: Opts = Opts::parse();
     opts.service.logging_format.init_subscriber();
 
+    tracing::info!("Version: {}", VERSION);
```

Comment:
If the service restarts, this doesn't get printed again right? I think it'd be better to print at every service restart since we often receive partial logs from users